core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -194,16 +193,9 @@ class KafkaConfig private(doLog: Boolean, val props: util.Map[_, _])
   private val _quorumConfig = new QuorumConfig(this)
   def quorumConfig: QuorumConfig = _quorumConfig
 
-  private val _groupCoordinatorConfig = new GroupCoordinatorConfig(this)
```

Comment:
What's the drawback of keeping them here? It would be inlined with the others, no?

Reply:
@dajac do you have any suggestions or concerns about moving `GroupCoordinatorConfig` out of `KafkaConfig`?